testing/configs/config_win10_1000_5000.yaml

```diff
@@ -0,0 +1,8 @@
+output: testing/output/btimevwebext_win10_1000_5000.json
+
+file_groups:
+  windows10:
+    task_group_id: HoEKn_W0R3eQ-sbVn8SJFQ
+    path: /home/vhyt/Documents/Mozilla/Perftest/fperf-analysis-win10-1000-5000
```

Comment:
Add this data folder into `testing/resources` as well.

Reply:
There are probably files in this folder that are not needed like `taskid-file-mapping.json` so you can exclude them (along with the `download` folders). We only need to have the data in the `perfherder-data_data` folders.